### PR TITLE
enh(chore) replace centcore by gorgone

### PR DESCRIPTION
**Pull request by loiclau**
## Description

I want Centreon Gorgone to be installed when installing Centreon Web
I want centcore to disappear forever from the source code

## Information
    Modify centreon-web SPEC template to include a dependance with centreon-gorgone
    Add missing perl-CryptX dep in repos. 
    Delete all centcore source code from Centreon project,
    Delete centcore source code references in SPEC templates.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: doc/fr/faq/remote_server.rst [hunk 1]
```diff
@@ -334,13 +334,7 @@ central.
...
 3. Fichiers manquants dans le répertoire /etc/centreon
 
-Vérifiez que le répertoire **/etc/centreon** contient les fichiers suivants :
-
-* instCentCore.conf
-* instCentPlugins.conf
-* instCentWeb.conf
-
-et que ces fichiers ne sont pas vides, sinon copiez les depuis le serveur Centreon
+Vérifiez que le fichier **/etc/centreon/instCentWeb.conf** n'est pas vide, sinon copiez les depuis le serveur Centreon
```
Comment (kduret) [inline doc/fr/faq/remote_server.rst]:
```suggestion
Vérifiez que le fichier **/etc/centreon/instCentWeb.conf** n'est pas vide, sinon copiez le depuis le serveur Centreon
```

File: doc/fr/faq/remote_server.rst [hunk 2]
```diff
@@ -334,13 +334,7 @@ central.
...
 3. Fichiers manquants dans le répertoire /etc/centreon
 
-Vérifiez que le répertoire **/etc/centreon** contient les fichiers suivants :
-
-* instCentCore.conf
-* instCentPlugins.conf
-* instCentWeb.conf
-
-et que ces fichiers ne sont pas vides, sinon copiez les depuis le serveur Centreon
+Vérifiez que le fichier **/etc/centreon/instCentWeb.conf** n'est pas vide, sinon copiez le depuis le serveur Centreon
```
Comment (sc979) [inline doc/fr/faq/remote_server.rst]: why this file is needed, when you seem to don't use it anymore ? 
https://github.com/centreon/centreon/pull/8172/files#diff-92e7108e0f126f93ae0f2687f4380febL1738

**Comment by loiclau** [comment]
An other branch was created for documentation: https://github.com/centreon/centreon/tree/MON-4505-update-doc-for-gorgone

File: doc/en/upgrade/from_sources.rst [hunk 3]
```diff
@@ -73,8 +73,7 @@ Load the previous installation parameters::
     [y/n], default to [y]:
     > y
 
-    Using:  /etc/centreon/instCentCore.conf
-    /etc/centreon/instCentPlugins.conf
+    Using:  /etc/centreon/instCentPlugins.conf
```
[the remark this answers is not in the document]
Comment (loiclau) [reply doc/en/upgrade/from_sources.rst]: instCentCore.conf

File: www/install/sql/centreon/Update-DB-20.04.0-beta.1.sql [hunk 4]
```diff
@@ -10,3 +10,6 @@ DELETE FROM `cb_type` WHERE `type_shortname` = 'correlation';
 UPDATE topology SET topology_url_opt = '&o=svcOV_pb' WHERE topology_page = 20204;
 UPDATE topology SET topology_url_opt = '&o=svcOVHG_pb' WHERE topology_page = 20209;
 UPDATE topology SET topology_url_opt = '&o=svcOVSG_pb' WHERE topology_page = 20212;
+
+DELETE FROM `options` WHERE `key` = 'enable_perfdata_sync';
```
Comment (kduret) [inline www/install/sql/centreon/Update-DB-20.04.0-beta.1.sql]: this does not need to be removed from fresh install ?

File: doc/en/faq/remote_server.rst [hunk 5]
```diff
@@ -323,13 +323,7 @@ Then generate the Remote Server configuration from the central Centreon server.
 
 3. Missing files in /etc/centreon
 
-Check that **/etc/centreon** includes the following files and that they are not empty:
-
-* instCentCore.conf
-* instCentPlugins.conf
```
[the remark this answers is not in the document]
Comment (lpinsivy) [reply doc/en/faq/remote_server.rst]: No only instCentWeb.conf is needed for modules and source update
I change all module to use only instCentWeb.conf (see associated PR for modules)

**Pull request merged**